main.py

```diff
@@ -49,26 +50,23 @@ def _make_layout(self) -> html.Div:
         config = self._config
         return make_page_layout(mesh_test_results, results_timestamp, config)
 
-
-def get_auth_email_token() -> Tuple[str, str]:
-    try:
-        email = os.environ["KTAPI_AUTH_EMAIL"]
-        token = os.environ["KTAPI_AUTH_TOKEN"]
-        return email, token
-    except KeyError:
-        logger.error("You have to specify KTAPI_AUTH_EMAIL and KTAPI_AUTH_TOKEN environment variables first")
-        sys.exit(1)
+    @staticmethod
+    def _get_auth_email_token() -> Tuple[str, str]:
+        try:
+            email = os.environ["KTAPI_AUTH_EMAIL"]
+            token = os.environ["KTAPI_AUTH_TOKEN"]
+            return email, token
+        except KeyError:
+            raise Exception("You have to specify KTAPI_AUTH_EMAIL and KTAPI_AUTH_TOKEN environment variables first")
```

Comment:
(non-critical) The exception text should say what condition has occurred, not what the user should do. I propose:
```
try
    return os.environ["KTAPI_AUTH_EMAIL"], os.environ["KTAPI_AUTH_TOKEN"]
except KeyError as e:
    raise RuntimeException(f'Missing required environment variable: {str(e}') 
```
For `KeyError` exception `str(e)` returns the missing key as string. There is always going to be only one, because the first missing environment variable triggers the exception. The local variables are not necessary.